index.html.md.erb

```diff
@@ -8,4 +8,5 @@ owner: API
 ## <a id='toc'></a> Table of Contents
 
 * [REST Overview](./rest-overview.html)
+* [OAuth Overview](./oauth-overview.html)
```

Comment:
Please include all sub-pages in the table of contents. Otherwise, there won't be anything highlighted in the navigation when the user is on one of these pages.